app/src/App.tsx

```diff
@@ -105,6 +106,16 @@ const App = () => {
 
         dispatch(updateTotalPeers(totalPeers));
       });
+
+      socket.on('latest_deploy', (latestDeployString: string) => {
+        const parsedLatestDeploy = JSON.parse(latestDeployString) as {
+          latestDeploy: ApiData.ProcessedSidecarDeploy;
+        }[];
```

Comment:
Would be awesome if we could get these types from a shared library or package :(